wrapper/ibus/setup/main.py

```diff
@@ -503,7 +503,10 @@ def __init_gettext(self):
         locale.setlocale(locale.LC_ALL, "")
         localedir = os.getenv("IBUS_LOCALEDIR")
         gettext.bindtextdomain(GETTEXT_PACKAGE, localedir)
-        gettext.bind_textdomain_codeset(GETTEXT_PACKAGE, "UTF-8")
+        try:
+            gettext.bind_textdomain_codeset(GETTEXT_PACKAGE, "UTF-8")
+        except AttributeError:
+            pass
```

Comment:
this API was deprecated in Python 3.8, and removed in Python 3.10. i'd prefer just removing it. but this change is also an improvement. so i will merge it.